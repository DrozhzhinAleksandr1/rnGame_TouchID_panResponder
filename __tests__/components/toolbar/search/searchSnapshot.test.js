import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { Search } from '/components/toolbar/search/search';

it('create snapshot Search', () => {
    const renderer = new ShallowRenderer()
    let SearchSnapshot = renderer.render(<Search />);
    expect(SearchSnapshot).toMatchSnapshot();
});