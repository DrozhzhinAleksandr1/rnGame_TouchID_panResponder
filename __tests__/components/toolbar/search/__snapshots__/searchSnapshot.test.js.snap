// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot Search 1`] = `
<TouchableWithoutFeedback
  accessibilityLabel="openSearchInputID"
  testID="openSearchInputID"
>
  <Component
    style={
      Object {
        "alignItems": "center",
        "height": "100%",
        "justifyContent": "center",
        "width": 56,
      }
    }
  >
    <Icon_search />
  </Component>
</TouchableWithoutFeedback>
`;
